### DISCUSSION TITLE
PDF Converter

### DISCUSSION THREAD

**Discussion by alberk8** [ariacom/Seal-Report#90] [Q&A]
From reading the documentation, it seems that a Windows platform is need to render PDF even if the Report Server resides on a Linux platform.   Why is it so?  Is it possible to run the whole setup in Linux?

**Comment by ariacom** [comment]
It is because the converter rely on a component that works only on Windows, thus you need a Windows Server as explained at https://ariacom.com/SealConverterInstallation

**Reply by alberk8** [reply]
Thank you for the reply.   Maybe in the future you can move to QuestPDF.  :)

**Comment by ariacom** [comment]
From version 8.0, QuestPDF is now used to generate PDF from Seal Report.
